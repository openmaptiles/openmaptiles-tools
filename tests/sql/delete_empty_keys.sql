-- delete_empty_keys
SELECT delete_empty_keys(NULL) AS v1_null;
SELECT delete_empty_keys(''::hstore) AS v2_null;
SELECT delete_empty_keys('"empty"=>""'::hstore) AS v3_null;
SELECT delete_empty_keys('"foo"=>"bar"'::hstore) AS v4;
SELECT delete_empty_keys('"foo"=>"bar", "empty"=>""'::hstore) AS v5;
SELECT delete_empty_keys('"foo"=>"bar", "empty"=>"", "xx"=>"zz"'::hstore) AS v6;
SELECT delete_empty_keys('"empty"=>"", "foo"=>"bar"'::hstore) AS v7;
SELECT delete_empty_keys('""=>"empty_key"'::hstore) AS v8;
SELECT delete_empty_keys('"nil"=>NULL, "foo"=>"bar"'::hstore) AS v9;
SELECT delete_empty_keys('"nil"=>NULL'::hstore) AS v10_null;
