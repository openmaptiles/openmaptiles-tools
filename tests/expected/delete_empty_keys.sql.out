 v1_null 
---------
 
(1 row)

 v2_null 
---------
 
(1 row)

 v3_null 
---------
 
(1 row)

      v4      
--------------
 "foo"=>"bar"
(1 row)

      v5      
--------------
 "foo"=>"bar"
(1 row)

            v6            
--------------------------
 "xx"=>"zz", "foo"=>"bar"
(1 row)

      v7      
--------------
 "foo"=>"bar"
(1 row)

       v8        
-----------------
 ""=>"empty_key"
(1 row)

      v9      
--------------
 "foo"=>"bar"
(1 row)

 v10_null 
----------
 
(1 row)

